### PR TITLE
Cleaning up contract code

### PR DESCRIPTION
**Pull request by adilanchian**
Wanted to give a bit of clean up here for our helper methods :)

### CODE REVIEW

File: .auth.config.sample.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  MUMBAI: {
+    BASE_TOKEN_URI: 'https://staging.tokens.buildspace.so/assets/',
```
Comment (assimovt) [inline .auth.config.sample.js:3]: nit: i'd not have trailing slash

File: contracts/scripts/utils.js [hunk 2]
```diff
@@ -31,110 +31,93 @@ async function main() {
   console.log('Token address:', token.address);
 }
 
-async function cancelTransaction(nonce) {
+// Helpers
+
+/**
+ * @param number nonce
+ * @param string gasPrice
+ * If a transaction is stuck, you can use this function to cancel that transaction as long as the gasPrice value is larger
```
Comment (assimovt) [inline contracts/scripts/utils.js:39]: nit: we should probly stick to the common format where first is the func summary, then params

File: contracts/scripts/utils.js [hunk 3]
```diff
@@ -31,110 +31,93 @@ async function main() {
...
 
-async function fetchCohort(cohortId) {
+/**
+ * @param string cohortId
+ * Fetches the cohort on the contract and prints out the Limit number of the cohort
+ */
+const fetchCohort = async (cohortId) => {
   const tx = await contract.cohorts(cohortId);
   console.log(tx);
   console.log('LIMIT:', tx[0].toNumber());
```
Comment (assimovt) [inline contracts/scripts/utils.js:61]: do we need these

File: contracts/scripts/utils.js [hunk 4]
```diff
@@ -31,110 +31,93 @@ async function main() {
...
-async function createCohort(cohort_id, limit) {
+};
+
+/**
+ * @param string cohort_id
+ * @param number limit
+ * @param string gasPrice
+ * Creates a new cohort on the contract with an initial merkle root
+ */
+const createCohort = async (cohort_id, limit, gasPrice) => {
```
Comment (assimovt) [inline contracts/scripts/utils.js:93]: consistency: camelCase used everywhere else, but not cohort_id

File: contracts/scripts/utils.js [hunk 5]
```diff
@@ -31,110 +31,93 @@ async function main() {
...
-
-createCohort('CHa07ac83b-bc08-46b6-b561-abe05d5bda00', 1000)
+};
+
+/**
+ * @param string address
+ * @param string cohort_id
+ * Fetches the token number of a given cohort_id and address
+ */
+const fetchOwner = async (address, cohort_id) => {
```
Comment (assimovt) [inline contracts/scripts/utils.js:112]: same here

File: contracts/scripts/utils.js [hunk 6]
```diff
@@ -31,110 +31,93 @@ async function main() {
...
+  const tx = await contract.claimed(address, cohort_id);
+  console.log(tx.toNumber());
+};
+
+// Execution Land
+createCohort('CHaee9050e-fdb6-4094-9097-7f4fa00b8fd6', 1000, '100')
```
Comment (assimovt) [inline contracts/scripts/utils.js:118]: hmm.. hard coded id?

File: .auth.config.sample.js (hunk 1 above)
Comment (adilanchian) [reply .auth.config.sample.js:3]: done!

File: contracts/scripts/utils.js (hunk 2 above)
Comment (adilanchian) [reply contracts/scripts/utils.js:39]: great catch ty!

File: contracts/scripts/utils.js (hunk 3 above)
Comment (adilanchian) [reply contracts/scripts/utils.js:61]: Yes essentially these funcs are for printing out info like this

File: contracts/scripts/utils.js (hunk 4 above)
Comment (adilanchian) [reply contracts/scripts/utils.js:93]: nice!

File: contracts/scripts/utils.js (hunk 6 above)
Comment (adilanchian) [reply contracts/scripts/utils.js:118]: This is just an example! Tweaked though :)

**Pull request merged**